packages/plugin-ext/src/main/browser/notebooks/renderers/cell-output-webview.tsx

```diff
@@ -117,7 +117,19 @@ export class CellOutputWebviewImpl implements CellOutputWebview, Disposable {
         }
 
         this.webviewWidget = await this.widgetManager.getOrCreateWidget(WebviewWidget.FACTORY_ID, { id: this.id });
-        this.webviewWidget.setContentOptions({ allowScripts: true });
+        this.webviewWidget.setContentOptions({
+            allowScripts: true,
+            // list taken from vscode backLayerWebview.ts:758
```

Comment:
Suggestion: Please use a commit link to make things unambiguous

https://github.com/microsoft/vscode/blob/a27099233b956dddc2536d4a0d714ab36266d897/src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts#L762-L774